index.html

```diff
@@ -667,24 +667,22 @@ <h2>
       </p>
 
       <p>
-A <dfn>conforming DID Method</dfn> is any specification that complies with the
+A <a>conforming DID method</a> is any specification that complies with the
 relevant normative statements in Section <a href="#methods"></a>.
       </p>
 
       <p>
-A <dfn>producer</dfn> is any algorithm realized as software and/or hardware and
-conforms to this specification if it generates <a>conforming DID</a>s or
-<a>conforming DID Document</a>s. A <a>producer</a> that is conformant with this
-specification MUST NOT produce non-conforming <a>DIDs</a> or <a>DID
-Documents</a>.
+A <a>conforming producer</a> is any algorithm realized as software and/or
+hardware and conforms to this specification if it generates <a>conforming
+DID</a>s or <a>conforming DID Document</a>s. A <a>conforming producer</a> MUST
+NOT produce non-conforming <a>DIDs</a> or <a>DID documents</a>.
       </p>
 
       <p>
-A <dfn>consumer</dfn> is any algorithm realized as software and/or hardware and
-conforms to this specification if it consumes <a>conforming DID</a>s or
-<a>conforming DID Document</a>s. A <a>consumer</a> that is conformant with this
-specification MUST produce errors when consuming non-conforming <a>DIDs</a> or
-<a>DID Documents</a>.
+A <a>conforming consumer</a> is any algorithm realized as software and/or
+hardware and conforms to this specification if it consumes <a>conforming
+DID</a>s or <a>conforming DID document</a>s. A <a>conforming consumer</a> MUST
+produce errors when consuming non-conforming <a>DIDs</a> or <a>DID documents</a>.
```

Comment:
Hmm, don't think I agree with moving conformance statements to terminology. I do understand that "conforming DID document" is a term we use throughout the specification, but it has more to do with conformance than it does with terminology. To put it another way, I think it's fine for certain things to only appear in the sections that define them if no other section points to them. The conformance statements fall into that category, IMHO. 

To put it another way, do we think that someone in some other group is going to pull in the "conforming DID document" definition into their spec? I hope not... I'm not sure it adds to another spec to refer to stuff as a "conforming this" or a "conforming that".

Reply:
I agree. I moved them all because the definition for DID resolver refers to "conforming DID document", so when the terms.html is pulled into other specs (ie. the use cases), the internal ref breaks there. Probably easiest just to remove the link around "conforming DID document" in the DID resolver definition and put them back?

Reply:
Updated to do [that](https://github.com/w3c/did-core/pull/321#discussion_r443184308) ^